tests/bagatom/test_bagToXML.py

```diff
@@ -0,0 +1,208 @@
+import os
+import pytest
+
+from mock import mock_open
+
+from codalib import bagatom
+
+
+# Settings for the monkeypatched fixtures
+TEST_NAME = 'baz'
+TEST_PATH = os.path.join('/foo/bar', TEST_NAME)
+FILE_COUNT = 3
+PAYLOAD_SIZE = 1500
+BAGGING_DATE = '2015-01-01'
+BAGIT_CONTENTS = 'BagIt-Version: 0.96\nTag-File-Character-Encoding: UTF-8'
+
+
+@pytest.fixture
+def bagxml(monkeypatch):
+    """
+    Test fixture that patches the following functions.
+        open()
+        bagatom.getBagTags()
+        bagatom.getOxum()
+    """
+    m = mock_open(read_data=BAGIT_CONTENTS)
+    monkeypatch.setattr('__builtin__.open', m)
+
+    monkeypatch.setattr('codalib.bagatom.getBagTags', lambda x: {})
+
+    monkeypatch.setattr(
+        'codalib.bagatom.getOxum',
+        lambda x: '{0}.{1}'.format(PAYLOAD_SIZE, FILE_COUNT))
+
+    return bagatom.bagToXML(TEST_PATH)
+
+
+@pytest.fixture
+def bagxml_with_tags(monkeypatch):
+    """
+    Test fixture that patches the following functions.
+        open()
+        bagatom.getBagTags()
+        bagatom.getOxum()
+
+    Returns getBagTags returns a predefined set of tags.
+    """
+    m = mock_open(read_data=BAGIT_CONTENTS)
+    monkeypatch.setattr('__builtin__.open', m)
+
+    bagtags = {
+        'Source-Organization': "Test Org",
+        'Bag-Size': '1GB'
+    }
+    monkeypatch.setattr('codalib.bagatom.getBagTags', lambda x: bagtags)
+
+    monkeypatch.setattr(
+        'codalib.bagatom.getOxum',
+        lambda x: '{0}.{1}'.format(PAYLOAD_SIZE, FILE_COUNT))
+
+    return bagatom.bagToXML(TEST_PATH)
+
+
+@pytest.fixture
+def bagxml_with_bagging_date(monkeypatch):
+    """
+    Test fixture that patches the following functions.
```

Comment:
In this and the previous test docstrings, I could be wrong, but shouldn't the period on this line actually be a colon?